Swift/LogSinks.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  LogSinks.swift
+//  CouchbaseLite
+//
+//  Copyright (c) 2025 Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import CouchbaseLiteSwift_Private
+
+/// A static container for managing the three log sinks used by Couchbase Lite.
+public class LogSinks {
+    /// The console log sink, enabled by default with a warning level.
+    public static var console: ConsoleLogSink? = .init(level: .warning) {
+        didSet {
+            if let console = self.console {
+                let level = CBLLogLevel(rawValue: UInt(console.level.rawValue))!
+                let domains = CBLLogDomain(rawValue: UInt(console.domains.rawValue))
+                CBLLogSinks.console = CBLConsoleLogSink(level: level, domains: domains)
+            } else {
+                CBLLogSinks.console = nil
+            }
+        }
+    }
+    
+    /// The file log sink, disabled by default.
+    public static var file: FileLogSink? = nil {
+        didSet {
+            if let file = self.file {
+                let level = CBLLogLevel(rawValue: UInt(file.level.rawValue))!
+                CBLLogSinks.file = CBLFileLogSink(level: level,
+                                                  directory: file.directory,
+                                                  usePlaintext: file.usePlaintext,
+                                                  maxKeptFiles: Int(file.maxKeptFiles),
```

Comment:
I think it should be Int64(file.maxKeptFiles)

Reply:
Int is correct as it's NSInteger not long long.